packages/wallets/src/components/Base/WalletCheckbox/WalletCheckbox.scss

```
@@ -0,0 +1,39 @@
+.wallets-checkbox {
+    display: flex;
+    justify-content: flex-start;
+    cursor: pointer;
+    align-items: center;
+    user-select: none;
+    -webkit-touch-callout: none;
+    -webkit-tap-highlight-color: transparent;
+
+    &__input {
+        display: none;
+    }
+    &__box {
+        display: flex;
+        width: 16px;
+        height: 16px;
+        transition: 0.3s ease-in-out;
+        flex-shrink: 0;
+        margin: 0 8px;
+        border-radius: 2px;
+        align-self: center;
+        box-sizing: border-box;
+        border: 2px solid var(--text-less-prominent);
+        outline: none;
```

small nit to match the styling convention:
```suggestion
    &__box {
        display: flex;
        width: 1.6rem;
        height: 1.6rem;
        transition: 0.3s ease-in-out;
        flex-shrink: 0;
        margin: 0 0.8rem;
        border-radius: 0.2rem;
        align-self: center;
        box-sizing: border-box;
        border: 0.2rem solid var(--text-less-prominent);
        outline: none;
```